src/audio/copier/copier_host.c

```diff
@@ -149,15 +149,28 @@ __cold int copier_host_create(struct comp_dev *dev, struct copier_data *cd,
 	config->type = SOF_COMP_HOST;
 	cd->gtw_type = ipc4_gtw_host;
 
-	audio_stream_fmt_conversion(copier_cfg->base.audio_fmt.depth,
-				    copier_cfg->base.audio_fmt.valid_bit_depth,
-				    &in_frame_fmt, &in_valid_fmt,
-				    copier_cfg->base.audio_fmt.s_type);
-
-	audio_stream_fmt_conversion(copier_cfg->out_fmt.depth,
-				    copier_cfg->out_fmt.valid_bit_depth,
-				    &out_frame_fmt, &out_valid_fmt,
-				    copier_cfg->out_fmt.s_type);
+	ret = audio_stream_fmt_conversion(copier_cfg->base.audio_fmt.depth,
+					  copier_cfg->base.audio_fmt.valid_bit_depth,
+					  &in_frame_fmt, &in_valid_fmt,
+					  copier_cfg->base.audio_fmt.s_type);
```

Comment:
should there be a notification to the host in case of such errors? We have them now, but should be rate-limited...